### ISSUE TITLE
System.DllNotFoundException: libuv

### ISSUE THREAD

**Issue by nutkracker** [lluppes/pisharp#4]
Having followed the install process up to step 5, replacing the install of kvm with dnvm... followed step 6... but when attempting to use ASP.NET 5 I receive an error with libuv.

lrwxrwxrwx 1 root root      18 May 27 09:36 libuv.so.1 -> libuv.so.1.0.0-rc1
-rwxr-xr-x 1 root root  419316 May 27 09:36 libuv.so.1.0.0-rc1

Welcome to MyNodes.NET. 
Please configure the system in the web interface.

System.DllNotFoundException: libuv
  at (wrapper managed-to-native) Microsoft.AspNet.Server.Kestrel.Networking.Libuv+NativeMethods:uv_loop_size ()
  at Microsoft.AspNet.Server.Kestrel.Networking.Libuv.loop_size () <0x6f73f560 + 0x00023> in <filename unknown>:0 
  at Microsoft.AspNet.Server.Kestrel.Networking.UvLoopHandle.Init (Microsoft.AspNet.Server.Kestrel.Networking.Libuv uv) <0x6f73f478 + 0x0002f> in <filename unknown>:0 
  at Microsoft.AspNet.Server.Kestrel.KestrelThread.ThreadStart (System.Object parameter) <0x6f73eef0 + 0x0008f> in <filename unknown>:0

**Comment by ghost** [comment]
I agree that Step 5 has changed (this stuff was set up a year ago with an old beta and an Pi 2).  
Did you run Step 6 - which installs libuv?

**Comment by nutkracker** [comment]
I'm waiting on MyNodes rebuild and will try again... Thanks!
